roles/openshift_master/tasks/main.yml

```diff
@@ -125,20 +145,23 @@
     content: "{{ openshift.master.request_header_ca }}"
     mode: 0600
     backup: yes
-  when: openshift.master.request_header_ca is defined and item.kind == 'RequestHeaderIdentityProvider' and item.clientCA | default('') != ''
+  when:
+  - openshift.master.request_header_ca is defined
+  - item.kind == 'RequestHeaderIdentityProvider'
+  - item.clientCA | default('') != ''
   with_items: "{{ openshift.master.identity_providers }}"
 
 # This is an ugly hack to verify settings are in a file without modifying them with lineinfile.
 # The template file will stomp any other settings made.
 - block:
-    - name: check whether our docker-registry setting exists in the env file
-      command: "awk '/^OPENSHIFT_DEFAULT_REGISTRY=docker-registry.default.svc:5000/' /etc/sysconfig/{{ openshift.common.service_type }}-master"
-      failed_when: false
-      changed_when: false
-      register: already_set
+  - name: check whether our docker-registry setting exists in the env file
```

Comment:
@tbielawa undoing your change. Even if the Ansible docs shows as correct, I don't like it.

[1] http://docs.ansible.com/ansible/latest/playbooks_blocks.html#id1